lib/mini-lightbox.js

```diff
@@ -120,6 +159,40 @@
     };
 
     var cache = MiniLightbox._cache = {};
-
-    root.MiniLightbox = MiniLightbox;
```

Comment:
We could just say here, `module.exports = MiniLightbox` and let `dist-it` (in fact Browserify) to take care of creating the global in case if it's needed.